.eslintrc.json

```diff
@@ -18,8 +18,7 @@
     "prettier/@typescript-eslint"
   ],
   "parserOptions": {
-    "ecmaVersion": 6,
-    "project": "./tsconfig.json"
+    "ecmaVersion": 6
```

Comment:
Removed due to repeated errors being showed while I tried to commit files in the test folder
`C:\bitgo\BitGoJS\modules\account-lib\test\unit\coin\hbar\keyPair.ts
0:0  error  Parsing error: "parserOptions.project" has been set for @typescript-eslint/parser.
The file does not match your project config: test\unit\coin\hbar\keyPair.ts.
The file must be included in at least one of the projects provided`

Reply:
I suspect that this is due to the definition done on the main eslint file that forces account-lib eslint file to define a parserOptions.project.

Reply:
oh interesting